Client/Frontend/Home/FxHomeHorizontalCell.swift

```diff
@@ -86,12 +86,8 @@ class FxHomeHorizontalCell: UICollectionViewCell, ReusableCell {
     private var fallbackFaviconBackground: UIView = .build { view in
         view.layer.cornerRadius = TopSiteCellUX.cellCornerRadius
         view.layer.borderWidth = TopSiteCellUX.borderWidth
-        view.layer.shadowOffset = CGSize(width: 0, height: 2)
```

Comment:
I think we should keep the shadow for the `fallbackFaviconBackground` because it highlights the image in the cell

Reply:
The direction in the ticket said the shadow should be removed, but I'll revert this part of the change.

Reply:
ohh sorry I didn't see the instructions to remove the shadow of the favIcon background. I'm sorry you had to revert it

Reply:
thank you 🚀